backend-services/autostarter/core/autostarter.py

```diff
@@ -0,0 +1,260 @@
+import os
+import re
+import socket
+import time
+
+import requests
+import ujson as json
+from artemis_utils import AUTO_RECOVER_PROCESS_STATE
+from artemis_utils import DB_HOST
+from artemis_utils import DB_NAME
+from artemis_utils import DB_PASS
+from artemis_utils import DB_PORT
+from artemis_utils import DB_USER
+from artemis_utils import get_logger
+from artemis_utils import TEST_ENV
+from artemis_utils.db_util import DB
+
+# logger
+log = get_logger()
+
+CHECK_INTERVAL = int(os.getenv("CHECK_INTERVAL", 5))
+SERVICE_NAME = "autostarter"
```

Comment:
would it make sense to have these as CONSTANTS in the `artemis_utils.constants` package?

Reply:
yes, will do this in the end (please keep the discussion open so that I do not miss it)

Reply:
will do in separate issue, the PR is gigantic (let's review and comment on it first, and as a second step we fix 'nits). I will mark it as ready when done.